core/src/main/java/com/klaytn/caver/utils/Convert.java

```diff
@@ -58,7 +70,10 @@ public enum Unit {
         KLAY("KLAY", 18),
         KKLAY("kKLAY", 21),
         MKLAY("MKLAY", 24),
-        GKLAY("GKLAY", 27);
+        GKLAY("GKLAY", 27),
+        KEI("kei", 0),
+        GKEI("Gkei", 9),
+        KLAY("KLAY", 18);
```

Comment:
please remove this line and add KAIA

Reply:
Oops, There was a typo.
I updated. 
Thank you